css/css-grid/abspos/grid-positioned-item-dynamic-change-006.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang=en class="reftest-wait">
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Grid positioned item dynamic change</title>
+<link rel="author" title="Filipp Riabchun" href="mailto:talpa@yandex.ru">
+<link rel="help" href="https://drafts.csswg.org/css-grid-1/#abspos" title="9. Absolute Positioning">
+<meta name="assert" content="This test checks that positioned items react to grid track size changes.">
+<style>
+.grid {
+  display: inline-grid;
+  grid-gap: 8px;
+  position: relative;
+}
+
+.green {
+  background: green;
+  width: 100px;
+  height: 100px;
+}
+
+.red {
+  position: absolute;
+  border: 3px solid red;
```

Comment:
Please avoid using red color, as it usually means something goes wrong. Please change it by "blue" or for example.